dpppt-backend-sdk/dpppt-backend-sdk-ws/src/test/java/org/dpppt/backend/sdk/ws/insertmanager/InsertManagerTest.java

```diff
@@ -7,18 +7,75 @@
 import ch.qos.logback.classic.Logger;
 import ch.qos.logback.classic.spi.ILoggingEvent;
 import ch.qos.logback.core.AppenderBase;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
 import java.time.Duration;
+import java.time.Instant;
 import java.util.ArrayList;
+import java.util.Base64;
 import java.util.List;
+import java.util.UUID;
 import org.dpppt.backend.sdk.model.gaen.GaenKey;
 import org.dpppt.backend.sdk.semver.Version;
 import org.dpppt.backend.sdk.utils.UTCInstant;
 import org.dpppt.backend.sdk.ws.insertmanager.insertionmodifier.OldAndroid0RPModifier;
 import org.dpppt.backend.sdk.ws.util.ValidationUtils;
 import org.junit.Test;
 import org.slf4j.LoggerFactory;
+import org.springframework.security.oauth2.jwt.Jwt;
 
 public class InsertManagerTest {
+
+  protected String createToken(boolean fake, UTCInstant expiresAt) {
+    Claims claims = Jwts.claims();
+    claims.put("scope", "exposed");
+    claims.put("onset", "2020-04-20");
```

Comment:
Shouldn't this be put to `now() - 3 days` or so? It's probably checked nowhere for the moment. But there might be a filter added at a later time, rejecting `onset` dates older than 14 days...

Reply:
Yes we should and yes we should add such a filter and probably write an integration test for it. @UBaggeler shall we make a Issue for it?